Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.Events.cs

```diff
@@ -33,6 +35,10 @@ public partial class Menu
         private bool _altHandled;
         private bool _isLostFocus = true;
         private Control _lastFocusElementBeforeMenu;
+        private double _x1;
```

Comment:
How about keeping track of TopLeft point and BottomRight point. Or just TopLeft and use ActualWidth/ActualHeight when needed